src/app/machines/components/TableCheckbox/TableCheckbox.tsx

```diff
@@ -0,0 +1,69 @@
+import type { ReactNode } from "react";
+
+import type { InputProps, PropsWithSpread } from "@canonical/react-components";
+import { Input } from "@canonical/react-components";
+import classNames from "classnames";
+import { useSelector, useDispatch } from "react-redux";
+
+import { actions as machineActions } from "app/store/machine";
+import machineSelectors from "app/store/machine/selectors";
+import type { SelectedMachines } from "app/store/machine/types";
+import type { RootState } from "app/store/root/types";
+
+export enum Checked {
+  Checked = "true",
+  Mixed = "mixed",
+  Unchecked = "false",
+}
+
+type Props = PropsWithSpread<
+  {
+    callId?: string | null;
+    extraClasses?: string;
+    inputLabel?: ReactNode;
```

Comment:
What do you think about making the `inputLabel` required for this component?

Looks like - in contrary to the general purpose `Input` from `@canonical/react-components` - this component isn't meant to be wrapped in a `label` element which would provide it.

Reply:
I think that would work if it wasn't for `AllCheckbox` which just has an `aria-label` (or at least it will do now that you've pointed out that it's missing :)).